src/time-picker/time-picker.jsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import WindowListenable from '../mixins/window-listenable';
```

Comment:
possibly because someone removed the function but forgot to remove the mixin O.o